### PR TITLE
Small Refactor

### PR DESCRIPTION
**Pull request by TonybynMp4**
* Added Config and locales
* Fixed the truck explosion sending it flying away
* fixed the cops being able to do the mission
Needs to fix the police alerts i couldn't figure it out since im alone and since the police alerts are really confusing to me (Pretty sure the Police doesn't receive alerts rn)

### CODE REVIEW

**Comment by milotiro95** [comment]
Perhaps my mistakes but all looks great no error server or client side and when i'm robber the truck i just cannot pose the bomb to explose the trunk and get the money. :/ No drawtext or target available except open the door with target.

**Comment by TonybynMp4** [comment]
> Perhaps my mistakes but all looks great no error server or client side and when i'm robber the truck i just cannot pose the bomb to explose the trunk and get the money. :/ No drawtext or target available except open the door with target.

It should give you a 'hint' in the top right corner of your screen telling you to press G when you're close to the truck's doors 🤔

**Comment by milotiro95** [comment]
Nop I don't have this i try fews times and i can launch the mission the truck spawn I'm killing the guards and after nothing happens :/. Actually working with the main version

**Comment by TonybynMp4** [comment]
Only time i had a similar problem was when I restarted the script and launched the mission right away, like waiting 30 seconds would make it work fine

File: server/main.lua [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
-	if copsOnDuty >= ActivePolice then
-		TriggerClientEvent("AttackTransport:Pozwolwykonac", _source)
-		xPlayer.Functions.RemoveMoney('bank', ActivationCost, "armored-truck")
-
-		OdpalTimer()
-    else
-		TriggerClientEvent('QBCore:Notify', _source, 'Need at least '..ActivePolice.. ' SASP to activate the mission.')
-    end
+	else
+		TriggerClientEvent('QBCore:Notify', _source, Lang:t('error.alreadyactive'))
```
Comment (NoobySloth) [inline server/main.lua:32]: TriggerClientEvent('QBCore:Notify', _source, Lang:t('error.AlreadyActive'))

**Review by NoobySloth — COMMENTED** [review]
Locales error

**Pull request closed without merging**